tests/html_glimmer/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,493 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`block-statement.hbs 1`] = `
+{{#block param hashKey=hashValue as |blockParam|}}
+  Hello
+{{/block}}
+
+{{#block almost80CharacterLongPositionalParamThatIsFirstAlmost80Chars helloWorldParam key=here}}
+{{/block}}
+
+{{#block param param param param param param param hashKey=hashValue as |blockParam|}}
+  Hello
+{{/block}}
+
+{{#block param param param param param param param hashKey=HashValue hashKey=hashValue}}
+  Hello
+{{/block}}
+
+{{#block param param param param param param param param param param param param param}}
+  Hello
+{{/block}}
+
+{{#block hashKey=HashValue hashKey=hashValue hashKey=HashValue hashKey=hashValue hashKey=HashValue}}
+  Hello
+{{/block}}
+
+{{#block}}
+  {{#block}}
+   hello
+  {{/block}}
+{{/block}}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+{{#block param hashKey=hashValue as |blockParam|}}
+  Hello
+{{/block}}
+{{#block
+  almost80CharacterLongPositionalParamThatIsFirstAlmost80Chars
+  helloWorldParam
+  key=here
+}}{{/block}}
+{{#block
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  hashKey=hashValue as |blockParam|
+}}
+  Hello
+{{/block}}
+{{#block
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  hashKey=HashValue
+  hashKey=hashValue
+}}
+  Hello
+{{/block}}
+{{#block
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+  param
+}}
+  Hello
+{{/block}}
+{{#block
+  hashKey=HashValue
+  hashKey=hashValue
+  hashKey=HashValue
+  hashKey=hashValue
+  hashKey=HashValue
+}}
+  Hello
+{{/block}}
+{{#block}}{{#block}}hello{{/block}}{{/block}}
+`;
+
+exports[`component.hbs 1`] = `
+<user-greeting @name="Ricardo" @greeting="Olá" />
+{{@greeting}}, {{@name}}!
+
+<button onclick={{action next}}>Next</button>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<user-greeting @name="Ricardo" @greeting="Olá"></user-greeting>
+{{@greeting}}
+,
+{{@name}}
+!
+<button onclick={{action next}}>
+  Next
+</button>
+`;
+
+exports[`concat-statement.hbs 1`] = `
+<div class="hello {{if goodbye true}}">
+  Hello
+</div>
+
+<div class="hello {{if goodbye true}} {{if goodbye false}} {{if goodbye true}} {{if goodbye false}} {{if goodbye true}}">
+  Hello
+</div>
+
+<a href="/{{url}}/{{url}}"></a>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div class="hello{{if goodbye true}}">
+  Hello
+</div>
+<div
+  class="hello
+    {{if goodbye true}}
+    {{if goodbye false}}
+    {{if goodbye true}}
+    {{if goodbye false}}
+    {{if goodbye true}}"
+>
```

Comment:
Does it respect the jsx-bracket-same-line option?

Reply:
No it doesn't. I think a ternary in a few places would fix that though. I didn't write any support for options! :(

Reply:
It would be great to support it :)

Adding a new language to prettier works in two phases. The first one is to print everything in a correct way, which you mostly did here. The second one is to make all the things than can be thrown at it look good.

You’re now entering the second phase :) So expect to do a lot of tweaks everywhere.

Reply:
I expected this! I'm really happy to do this part too. I figured that it would need tweaking because a lot of the decisions I made were based off asking a few people in the community and then picking from what is available. I will be in airplane for a few hours next week so I'll have plenty of time to implement whatever is decided upon.